### DISCUSSION TITLE
文档搜索报错

### DISCUSSION THREAD

**Discussion by jarlemon** [element-plus/element-plus#3851] [Q&A] [answered]
https://element-plus.gitee.io/zh-CN/ 

这个网站上搜索时报错了是什么问题呢
![image](https://user-images.githubusercontent.com/30827091/136884112-e38b97dd-5bc4-4358-8178-3f2a4c960052.png)

**Comment by sxzz** [comment]
#3909

**Comment by sxzz** [comment, marked as answer]
Fixed #4004
